tests/draw/test_plt.py

```diff
@@ -15,7 +15,7 @@
 from tyssue import Sheet, config
 from tyssue.draw.plt_draw import quick_edge_draw, sheet_view
 from tyssue.draw.plt_draw import _face_color_from_sequence
-from tyssue.draw.plt_draw import create_gif
+from tyssue.draw.plt_draw import create_gif, plot_forceœ
```

Comment:
oups there's a typo

Reply:
I just fix it, the magic of key combination, I really don't know how to do this letter...